addons/TBMod_arsenal/functions/fn_loadDefault.sqf

```diff
@@ -11,19 +11,19 @@ if (_arsenalType == "USA") exitWith
-        case "arzt": {[[],[],[],["rhs_uniform_g3_tan",[["ACE_tourniquet",4],["ACE_Morphine",5],["ACE_CableTie",1],["ACE_MapTools",1],["ACE_packingBandage",15],["ACE_quikclot",15],["ACE_fieldDressing",10]]],["rhsusf_spcs_ocp_medic",[["ACE_surgicalKit",1],["ACE_personalAidKit",3],["ACE_tourniquet",4],["ACE_elasticBandage",50],["ACE_epinephrine",10]]],["B_Kitbag_cbr",[["ACE_Atropine",20],["ACE_bloodIV",4],["SmokeShell",6,1],["SmokeShellBlue",1,1]]],"rhsusf_cvc_alt_helmet","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]};
-        case "rifle" : {[[],[],[],["rhs_uniform_cu_ocp",[["ACE_MapTools",1],["ACE_CableTie",1],["ACE_tourniquet",4],["ACE_fieldDressing",15],["ACE_Atropine",10]]],["rhsusf_spcs_ocp",[["ACE_fieldDressing",15],["ACE_Atropine",5],["rhs_mag_mk3a2",3,1],["rhs_mag_m67",3,1],["SmokeShellBlue",1,1]]],["rhsusf_assault_eagleaiii_ocp",[["ACE_EntrenchingTool",1],["SmokeShell",6,1]]],"rhsusf_ach_helmet_ocp","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]};
-        case "pionier" : {[[],[],[],["rhs_uniform_cu_ocp",[["ACE_MapTools",1],["ACE_CableTie",1],["ACE_tourniquet",4],["ACE_fieldDressing",15],["ACE_Atropine",10]]],["rhsusf_spcs_ocp",[["ACE_fieldDressing",15],["ACE_Atropine",5],["rhs_mag_mk3a2",1,1],["rhs_mag_m67",2,1],["SmokeShellBlue",1,1]]],["rhsusf_assault_eagleaiii_ocp",[["ACE_EntrenchingTool",1],["SmokeShell",4,1]]],"rhsusf_ach_helmet_ocp","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]};
+        case "sani": {[[],[],[],["rhs_uniform_cu_ocp",[["ACE_surgicalKit",1],["ACE_quikclot",25]]],["rhsusf_spcs_ocp_medic",[["ACE_elasticBandage",40],["ACE_packingBandage",40],["ACE_epinephrine",10],["ACE_atropine",15],["ACE_morphine",10],["ACE_adenosine",15],["ACE_tourniquet",10]]],["B_Kitbag_mcamo",[["ACE_plasmaIV",10],["ACE_plasmaIV_500",10],["ACE_plasmaIV_250",5],["adv_aceCPR_AED",1]]],"","",["m24_bino","","","",[],[],""],["ItemMap","ItemGPS","TFAR_anprc152_6","ItemCompass","ACE_Altimeter",""]]};
+        case "mg": {[["rhs_weap_m249_pip_L","rhsusf_acc_nt4_black","rhsusf_acc_anpeq15A","rhsusf_acc_ACOG_RMR",["rhsusf_200rnd_556x45_mixed_box",200],[],""],[],[],["rhs_uniform_cu_ocp",[["ACE_fieldDressing",25],["ACE_adenosine",2],["ACE_atropine",9],["ACE_tourniquet",4]]],["rhsusf_spcs_ocp_machinegunner",[["ACE_EntrenchingTool",1],["ACE_MapTools",1],["ACE_Flashlight_KSF1",1],["rhsusf_200rnd_556x45_mixed_box",3,200],["SmokeShell",3,1]]],["rhsusf_assault_eagleaiii_ocp",[["ACE_salineIV",4],["rhsusf_200rnd_556x45_mixed_box",3,200],["SmokeShell",1,1]]],"rhsusf_ach_helmet_ocp","",["m24_bino","","","",[],[],""],["ItemMap","ItemGPS","TFAR_anprc152_8","ItemCompass","ACE_Altimeter",""]]};
```

Comment:
Bin eher dafür das man keine Waffen in den Default Loadouts drin sein sollten. Andere Meinungen?

Reply:
Es geht nur darum um den  Ablauf zu verschnellern für Leute die nicht wissen was sie nehmen sollen. Sie können immer noch Ihre eigene Ausrüstung daraus basteln.

Reply:
Bei der BW werde ich keine Waffen geben wenn ich da den rüstoß sehe bei den 5.56 Waffen (von NIArsenal) uffff...

Reply:
Die letzten DLs der USA hat @mordl gemacht, mit genau @Er1807 s Hintergedanken.
Bin unschlüssig, fand schon damals das es erfahrenden hilft (man muss nicht alles erst wegpacken usw) aber neuen ungemein hinderlich ist (brauchen mehrer Minuten mehr Ausrüstungszeit)

Reply:
DLs sind für die Neulinge. Jeder Erfahrene weiß wofür der "Layout abspeichern" Knopf steht und sollte diesen auch nutzen+eigene wenige missionsabhängige Anpassungen. Ich verwende nie ein DL, sodass ich für die Neuen eine Lanze hiermit breche.